RevenueCatUI/CustomerCenter/Views/CompatibilityLabeledContent.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  CompatibilityLabeledContent.swift
+//
+//
+//  Created by Facundo Menzella on 14/1/25.
+//
+
+import SwiftUI
+
+#if os(iOS)
+
+/// A SwiftUI view for displaying labeled content
+@available(iOS 15.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+struct CompatibilityLabeledContent<Label: View, Content: View>: View {
+
+    let label: () -> Label
+    let content: (() -> Content)?
```

Comment:
In `LabeledContent`, I don't think the `content` parameter can be `nil`. Does it have to be optional in `CompatibilityLabelContent`?

Reply:
Agreed. There's no point in having a labeled content without content 😅